pkg/webui/components/status/index.js

```diff
@@ -54,6 +55,7 @@ const Status = React.forwardRef(
       [style.statusBad]: status === 'bad',
       [style.statusMediocre]: status === 'mediocre',
       [style.statusUnknown]: status === 'unknown',
+      [style.statusNone]: status === 'none',
```

Comment:
Please remove the additions to the status component and see my other comment on using a plain `<Message />` instead.